components/org.wso2.carbon.identity.conditional.auth.functions.session/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/session/function/IsWithinSessionLimitFunction.java

```diff
@@ -157,6 +154,8 @@ private int getActiveSessionCount(AuthenticatedUser authenticatedUser) throws Fr
                     return sessionCount;
                 } catch (IOException e) {
                     throw new FrameworkException("Problem occurred while processing the HTTP Response ");
+                } catch (NumberFormatException e){
+                    throw new FrameworkException("Problem occurred while parsing response result ");
```

Comment:
include 'e', do the same for all other catch blocks

Reply:
fixed in the new commits